docs.json

```diff
@@ -332,6 +332,16 @@
         }
       ]
     },
+    {
+      "tab": "cloud",
+      "group": "Security",
+      "pages": [
+        {
+          "title": "Security Overview",
```

Comment:
**suggestion:** the page itself can be named "Overview" instead of "Security Overview"